chapters/emergencystop.adoc

```diff
@@ -0,0 +1,29 @@
+== Emergency stop
+
+This rule only applies to Division A.
+
+.Definition
+
+A team can ask to stop the game immediately after a grace period of 10 seconds or at the next stoppage, whichever happens first regardless of the current situation. It will receive a yellow card for this and must take a timeout immediately. If the team is out of timeouts, it is still allowed to remove robots from the field, but can not use any remaining timeout time.
```

Comment:
An info box explaining the rational of this rule might be a good idea. Stating that this is intended for fatal situations, e.g. where robots damage themselves.